apps/condo/domains/user/schema/CheckUserExistenceService.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Generated by `createservice user.CheckUserExistenceService --type queries`
+ */
+
+const get = require('lodash/get')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { checkDvAndSender } = require('@open-condo/keystone/plugins/dvAndSender')
+const { GQLCustomSchema, find } = require('@open-condo/keystone/schema')
+
+const { NOT_FOUND, WRONG_FORMAT, DV_VERSION_MISMATCH } = require('@condo/domains/common/constants/errors')
+const { normalizePhone } = require('@condo/domains/common/utils/phone')
+const access = require('@condo/domains/user/access/CheckUserExistenceService')
+const { ConfirmPhoneAction } = require('@condo/domains/user/utils/serverSchema')
+
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const ERRORS = {
+    MULTIPLE_USERS_FOUND: {
+        query: 'checkUserExistence',
+        code: NOT_FOUND,
+        type: 'MULTIPLE_USERS_FOUND',
+        message: 'Multiple users found',
+    },
+    INVALID_PHONE_NUMBER: {
+        query: 'checkUserExistence',
+        code: BAD_USER_INPUT,
+        type: 'INVALID_PHONE_NUMBER',
+        message: 'Invalid phone number',
+    },
+    TOKEN_NOT_FOUND: {
+        query: 'checkUserExistence',
+        variable: ['data', 'token'],
+        code: NOT_FOUND,
+        type: 'TOKEN_NOT_FOUND',
+        message: 'Token not found',
+    },
+    DV_VERSION_MISMATCH: {
+        query: 'checkUserExistence',
+        variable: ['data', 'dv'],
+        code: BAD_USER_INPUT,
+        type: DV_VERSION_MISMATCH,
+        message: 'Wrong value for data version number',
+    },
+    WRONG_SENDER_FORMAT: {
+        query: 'checkUserExistence',
+        variable: ['data', 'sender'],
+        code: BAD_USER_INPUT,
+        type: WRONG_FORMAT,
+        message: 'Invalid format of "sender" field value',
+        correctExample: '{ dv: 1, fingerprint: \'example-fingerprint-alphanumeric-value\'}',
+    },
+}
+
+const CheckUserExistenceService = new GQLCustomSchema('CheckUserExistenceService', {
+    types: [
+        {
+            access: true,
+            type: 'input CheckUserExistenceInput { dv: Int!, sender: JSON!, confirmActionToken: String!, userType: UserTypeType! }',
```

Comment:
```suggestion
            type: 'input CheckUserExistenceInput { dv: Int!, sender: SenderFieldInput!, confirmActionToken: String!, userType: UserTypeType! }',
```

Reply:
Token must be passed as` ConfirmPhoneActionWhereUniqueIUnput` or as `ID`

Reply:
```suggestion
            type: 'input CheckUserExistenceInput { dv: Int!, sender: SenderFieldInput!, confirmActionToken: ID!, userType: UserTypeType! }',
```

Reply:
> Token must be passed as` ConfirmPhoneActionWhereUniqueIUnput` or as `ID`

Token is not `ConfirmPhoneActionWhereUniqueIUnput`. It is simple string, type changed to `ID`.

Description from GraphQL docs
> The ID scalar type represents a unique identifier, often used to refetch an object or as the key for a cache. The ID type is serialized in the same way as a String; however, defining it as an ID signifies that it is not intended to be human‐readable.

So there is almost no difference